### ISSUE TITLE
[IMPROVE] Multiinstance identification of pep instances ps

### ISSUE THREAD

**Issue by cesarjorgemartinez** [telefonicaid/fiware-pep-steelskin#306]
ps -efa|grep pep
pepproxy  8313     1  0 10:08 ?        00:00:00 node /opt/pepProxy/bin/pepProxy
pepproxy  8361     1  0 10:08 ?        00:00:00 node /opt/pepProxy/bin/pepProxy

We can use something else STH:
ps -efa|grep sth
sth      15134     1  3 10:14 ?        00:00:00 /usr/bin/node /opt/sth/bin/sth /opt/sth/conf/sth_sth.conf

**Comment by dmoranj** [comment]
Please add some more text to the description, I can barely understand what you are talking about. Do you want me to change how PEP instances look in a linux `ps` command? Is that even possible?

**Comment by cesarjorgemartinez** [comment]
Yes, we can identify different instances if in the start add the name of the instance, or path to instance configuration.
In STH we have:
https://github.com/telefonicaid/fiware-sth-comet/blob/master/rpm/SOURCES/etc/init.d/sth
       su ${COMPONENT_USER} -p -c "${NODE_EXEC} ${COMPONENT_DIR}/bin/sth ${instance} 2>&1 | tee -a ${LOG_DIR}/${NAME}.log & echo \$! > ${STH_PID_FILE}"
